dev_guide/application_lifecycle/new_app.adoc

```diff
@@ -575,20 +568,16 @@ spec:
 ----
 <1> Including *builder* here ensures this `ImageStreamTag` appears in the
 web console as a builder.
-====
 
 . Modify the settings in the new application screen to configure the objects
 to support your application:
 +
-====
-
-image::create_from_image.png["Create from source"]
-====
+image::3.7-create-app-settings.png["Create from source"]
```

Comment:
This image what a user would see if she took the Advanced Options in the wizard. The default is to have this configuration step in a wizard.

Reply:
+1 Cameron, I was going to note the same thing here.

Reply:
@ahardin-rh I'll capture an animated gif of going through the wizard steps.